pkg/cli/admin/release/mirror.go

```diff
@@ -525,17 +501,23 @@ func (o *MirrorOptions) Run() error {
 
 	to := o.ToRelease
 	if len(to) == 0 {
-		to = targetFn("").Exact()
+		to = targetFn("").Ref.Exact()
 	}
-	if hasPrefix {
-		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirror prefix: %s\n", to, o.To)
-	} else {
-		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
+	fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\n", to)
+	if len(o.To) > 0 {
+		if hasPrefix {
+			fmt.Fprintf(o.Out, "Mirror prefix: %s\n", o.To)
+		} else {
+			fmt.Fprintf(o.Out, "Mirrored to: %s\n", o.To)
+		}
 	}
-
-	if o.PrintImageContentInstructions {
-		if err := printImageContentInstructions(o.Out, o.From, o.To, repositories); err != nil {
-			return fmt.Errorf("Error creating mirror usage instructions: %v", err)
+	if len(o.ToDir) > 0 {
+		fmt.Fprintf(o.Out, "\nTo upload local images to a registry, run:\n\n    oc image mirror --from-dir=%s file://%s* REGISTRY/REPOSITORY\n\n", o.ToDir, to)
+	} else if len(o.To) > 0 {
+		if o.PrintImageContentInstructions {
```

Comment:
should we move this to conditional block https://github.com/openshift/oc/pull/126/commits/a8c5d8f8c5cce6c81675e360d37f42fffeb3b9e1#diff-6c5f7c1cbfa2abdaa2847f0982a31e5fR507 ??